### ISSUE TITLE
Failure messages could be improved

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/echidna#129]
Tried to publish, but I initially got failure:The [error I got back from submitting](https://lists.w3.org/Archives/Public/public-tr-notifications/2015Mar/0005.html) is: 

```
    {
      "time": "2015-03-17T18:10:45.112Z",
      "fact": "You are not authorized to publish"
    },
```

Would be good if the service would tell you what you need to do to get authorization? @plehegar fixed it for me, but not sure what he did to get it to work.

**Comment by astorije** [comment]
@marcoscaceres, yes, I intend to provide better error report. That's the cost of an early release version :-)
I leave this open until you feel feedback is up to your expectations.

**Comment by marcoscaceres** [comment]
Thanks @astorije! Happy to review messages, etc. Here is another example from today where it's not clear how to proceed: https://lists.w3.org/Archives/Public/public-tr-notifications/2015Mar/0011.html

**Comment by astorije** [comment]
@marcoscaceres, I agree it must seem very cryptic indeed.

In this case, one of our dependencies (the [spec generator](https://github.com/w3c/spec-generator) has crashed, ouch...

**Comment by marcoscaceres** [comment]
Yeah, it might be useful to have a way to also check Echidna's status... I've been periodically sending publication requests through out the day hoping that it would start working again. You can see all my fails:

https://lists.w3.org/Archives/Public/public-tr-notifications/2015Mar/

**Comment by marcoscaceres** [comment]
Got another strange one: latest-is-not-previous 

https://lists.w3.org/Archives/Public/public-tr-notifications/2015May/0011.html

**Comment by tripu** [comment]
@marcoscaceres: yes, definitely it's bad UI now&hellip; We shall make it more legible very soon. Thank you for keeping track here.

In the meantime, when you see error *keys* specifically from Specberus, like this one, you can run your doc [through Specberus](https://labs.w3.org/pubrules/) to (sometimes) get a better error message. (In fact, for now I suggest that you always do that manually until Specberus returns zero errors, before submitting to Echidna.) In this case, [I see one error, but a different one, in Specberus right now](https://labs.w3.org/pubrules/?url=https%3A%2F%2Flabs.w3.org%2Fspec-generator%2F%3Ftype%3Drespec%26url%3Dhttps%3A%2F%2Fw3c.github.io%2Fmanifest%2Findex.html%3FspecStatus%3DWD%3BshortName%3Dappmanifest&profile=WD-Echidna&validation=simple-validation&noRecTrack=false&informativeOnly=false&echidnaReady=true&patentPolicy=pp2004&processDocument=2014)&hellip; (The *key* `latest-is-not-previous` in Specberus would return the message *&ldquo;retrieved &lsquo;previous&rsquo; and &lsquo;latest&rsquo; documents, but their contents don't match&rdquo;*.)
